Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -191,7 +196,9 @@ def update_step_view(self):
 
         self.title_label.setText(self.current_step.get_title())
         self.content_area.setText(self.current_step.get_content())
-
+        self.step_label.setText("Click next to continue.")
+        if self.current_step_index > 0:
+            self.step_label.setText(f"Step {self.current_step_index} of {self.current_tutorial_num_steps}")        
```

Comment:
Can move the `self.step_label.setText("Click next to continue.")` line until an `else` condition here since for all but the first step it will get overwritten with the `Step X of N`